modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeStatistics.pm

```diff
@@ -242,6 +248,21 @@ sub _get_number_of_proteins_used {
         $self->param( 'number_of_proteins_in_single_species_trees', $number_of_proteins_in_single_species_trees );
     }
     $sth->finish();
+
+    # Compute the number of genes that are unassigned to a tree
+    my $sql2 = q/
+        SELECT SUM(stnt.value)
+        FROM species_tree_node_tag stnt JOIN species_tree_node USING (node_id)
+        WHERE stnt.tag = 'nb_genes_unassigned'
+        AND node_id IN (SELECT node_id FROM species_tree_node WHERE genome_db_id IS NOT NULL)
```

Comment:
```suggestion
        SELECT 
            SUM(stnt.value)
        FROM
            species_tree_node_tag stnt
                JOIN
            species_tree_node USING (node_id)
        WHERE
            stnt.tag = 'nb_genes_unassigned'
                AND node_id IN (SELECT 
                    node_id
                FROM
                    species_tree_node
                WHERE
                    genome_db_id IS NOT NULL)
```
This is a **formatting** (not mandatory) suggestion to improve readability and make changing the SQL code if required in the future more visible to review with regards to fewer line changes if another conditional was added or one changed. To be considered for all SQL code.